### ISSUE TITLE
Error when using 'conan info .' (Conan 1.12.0)

### ISSUE THREAD

**Issue by climblinne** [conan-io/conan#4443]
To help us debug your issue please explain:

- [x] I've read the [CONTRIBUTING guide](https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md).
- [x] I've specified the Conan version, operating system version and any tool that can be relevant.
- [x] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

I tested Conan 1.12.0 and got the following error:

```
conan info .
Traceback (most recent call last):
  File "c:\tools\python3\lib\site-packages\conans\client\command.py", line 1495, in run
    method(args[0][1:])
  File "c:\tools\python3\lib\site-packages\conans\client\command.py", line 555, in info
    build=args.dry_build)
  File "c:\tools\python3\lib\site-packages\conans\client\conan_api.py", line 88, in wrapper
    return f(*args, **kwargs)
  File "c:\tools\python3\lib\site-packages\conans\client\conan_api.py", line 613, in info
    settings, options, env)
  File "c:\tools\python3\lib\site-packages\conans\client\conan_api.py", line 579, in _info_args
    self._cache, self._user_io.out)
  File "c:\tools\python3\lib\site-packages\conans\client\conan_api.py", line 971, in get_graph_info
    graph_info = GraphInfo.load(install_folder)
  File "c:\tools\python3\lib\site-packages\conans\model\graph_info.py", line 27, in load
    return GraphInfo.loads(load(p))
  File "c:\tools\python3\lib\site-packages\conans\model\graph_info.py", line 41, in loads
    root = graph_json["root"]
KeyError: 'root'

ERROR: 'root'
```

It works under 0.11.2.

This happens on a library with the following conanfile:

```
from conans import python_requires

base = python_requires("base_library/0.1@demo/testing")

class mytest_Conan(base.BaseLibrary):
    name = "mytest"
    url = "something"
    description = "something"
    scm = base.BaseLibrary.get_git_scm_data()
    version = base.BaseLibrary.get_git_version()
```
No dependent package in the requirements.

**Comment by climblinne** [comment]
Yes, thanks with `conan install .` it works. Thanks

**Comment by jgsogo** [comment]
I've opened a PR handling this exception to add better error message, with that change it should have been something like:

> ERROR: Error parsing GraphInfo from file '<your/path/to>/graph_info.json': Error related to GraphInfo text content: 'root'

**Comment by lasote** [comment]
Anyway, it shouldn't crash. We will try to fix it in the 1.12.1 minor.

**Comment by memsharded** [comment]
Fixed, it won't crash anymore, will be released in 1.12.1
